### PR TITLE
add ecosystem section

### PR DESCRIPTION
**Pull request by marbar3778**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,12 @@ If you are working on a module that you would like to be used by the community,
 - [Kava - CDP System](https://github.com/Kava-Labs/kava-devnet/tree/master/blockchain/x)
 - [Kava - Payment Channels](https://github.com/Kava-Labs/cosmos-paychan)
 
+## Ecosystem
+
+Projects built using the Cosmos-SDK
+
+- Cosmos Hub [Github](https://github.com/cosmos/gaia)
+
```
Comment (okwme) [inline README.md]:
```suggestion
 - Binance Chain [Github](https://github.com/binance-chain/)
 - e-Money [Website](https://e-money.com/)
 - IOV [Website](https://iov.one/)
 - IRISnet [Website](https://www.irisnet.org/)
 - Kava [Website](https://kava.io/)
 - Lino Network [Website](https://lino.network/)
 - Loom [Website](https://loomx.io/)
 - Playlist [Website](https://www.playlist.com/)
 - Regen Network [Website](https://regen.network/)
 - Sentinel [Website](https://sentinel.co/)
 - Terra [Website](https://terra.money/)
 - Trustory [Website](https://www.trustory.io/)
 - [More Here](https://forum.cosmos.network/t/list-of-projects-in-cosmos-tendermint-ecosystem/243) 
```

Comment (marbar3778) [reply README.md]: do you think website is better than github? or just on a per-entry basis?

I think we will have to move the forum thing over to here and the tendermint users to tendermint awesome

Comment (okwme) [reply README.md]: I just copied what was linked on https://cosmos.network/ecosystem, not sure if many of those projects have public githubs or not...

Comment (okwme) [reply README.md]: i'm not sure who maintains the forum, i'm a little hesitant to fix it if it ain't broke, but idk how active they've been in maintaining it. I'd hate to think the community stopped updating it if we moved it away from the original location, ya know?

Comment (marbar3778) [reply README.md]: Asomadt is maintaining it, we would have to give him write access to this repo. If we do transfer over

Comment (marbar3778) [reply README.md]: Also the topic is at its limit for additions so he is either starting a v2 or we suggest this place

**Comment by okwme** [comment]
Let's alphabetize it!

File: Contributing.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
 # Contributing
 
-Make a pull request against master. If the Pull request is incomplete, you can still open it, but under using the Draft PR feature of Github. 
-
-We encourage everyone building in the ecosystem to make contributions to this repo. 
+Make a pull request against master. If the Pull request is incomplete, you can still open it, but under using the Draft PR feature of Github.
```
Comment (okwme) [inline Contributing.md]:
```suggestion
Make a pull request against master. If the pull request is incomplete, you can still open it, but using the Draft PR feature of Github.
```

File: Contributing.md [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
+If you would like to be included in the wallet and/or block explorer section then there a few requirements.
 
 - Https - wallet, block explorers
 - Publication or have spoken about security audit - wallet
 - Twitter or Github to contact the developers - wallet, block explorer
 
+## Ecosystem
 
-## Modules
+If you would like to be added to the ecosystem section, please find your place in the list, its alphabitized.
```
Comment (okwme) [inline Contributing.md]:
```suggestion
If you would like to be added to the ecosystem section, please find your place in the list alphabetically.
```

**Pull request merged**